pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "openpipe-art"
-version = "0.1.25"
+version = "1.0.0"
```

Comment:
I assume this is a typo. I wouldn't bump to 1.0.0, not for a while 😅

Reply:
No, I just incremented the major version because you pointed out this PR includes a breaking change. Did you have something else in mind? I think at this stage, since we're in alpha/beta, it's fine not to follow semver strictly.

Reply:
Yeah, I just feel like anything >= 1.0.0 has expectations of being production ready. I'd just bump to `0.2.0`. I think most package managers pin major and minor versions and only allow the patch number to automatically upgrade, so I believe this should be okay.